eclipse-che/che-plugin-registry
# Update the vale version in the sidecar image

Signed-off-by: Igor Vinokur <ivinokur@redhat.com>



### What does this PR do?
Update the vale version in the sidecar image

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?



### How to test this PR?

1. Go to `sidecars/vale` directory.
2. Run `docker build .` 
3. Run `docker run <image id from the previous command> -it sh`
4. Run `vale --v`
See: the vale version is updated to `v2.10.1`

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
